src/modules/ekf2/EKF/velocity_fusion.cpp

```diff
@@ -79,12 +82,61 @@ void Ekf::updateVelocityAidSrcStatus(const uint64_t &time_us, const Vector3f &ob
 	aid_src.timestamp_sample = time_us;
 }
 
+void Ekf::fuseBodyVelocity(estimator_aid_source3d_s &aid_src)
+{
+	Vector3f innov;
+	Vector3f innov_var;
+	matrix::Matrix<float, 1, 23UL> H;
+	bool fused[3] = {false, false, false};
+	const auto state_vector = _state.vector();
+	Vector3f R(aid_src.observation_variance);
+
+	sym::ComputeEvBodyVelVarAndHx(state_vector, P, &innov_var, &H);
+	innov = _R_to_earth.transpose() * _state.vel - Vector3f(aid_src.observation);
+	innov.copyTo(aid_src.innovation);
+	(innov_var + R).copyTo(aid_src.innovation_variance);
```

Comment:
Then it's not innov_var if it requires `+R` to be the innovation variance

Reply:
true, I'll change it to `body_vel_var` as suggested in the derivation.py